Lib/test/test_statistics.py

```diff
@@ -2178,7 +2178,7 @@ def f(x):
     def test_specific_cases_inclusive(self):
         # Match results computed by hand and cross-checked
         # against the PERCENTILE.INC function in MS Excel
-        # and against the quaatile() function in SciPy.
+        # and against the quantile() function in SciPy.
```

Comment:
Wouldn't be better to reference NumPy instead of SciPy?
ref: https://docs.scipy.org/doc/numpy/reference/generated/numpy.quantile.html

Reply:
I will leave it to @rhettinger on changing scipy to numpy